props/saber_shtok_buttons.h

```diff
@@ -153,6 +240,69 @@ SaberShtokButtons() : PropBase() {}
         }
         return true;
 
+// Battle Mode
+
+#ifndef FETT263_BATTLE_MODE_ALWAYS_ON
```

Comment:
You need to rename these defines.
Your prop file should be using SHTOK_* instead of FETT263_*
Also, it would be helpful to list them at the top of the file together with a description what they do.

Reply:
I don't use these defines so I don't need them.

Reply:
If you don't want them, then you should delete them (and the code that is associated with it)

Reply:
Deleted in this new version.